code/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -38,30 +56,71 @@
    * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   @Inject
-  private RobotContainer(final ILogger logger, IDrivetrainSubsystem drivetrain,
-      DrivetrainControllerCommand driveCommand, IIntakeSubsystem intake, DriveArmCommand driveArmCommand,
-      IClimbSubsystem climb, ClimbControllerCommand climbCommand, Provider<Easy23StrategyCommand> autonomousCommand) {
+  private RobotContainer(final ILogger logger, RobotConfig config, IDrivetrainSubsystem drivetrain,
+      DrivetrainControllerCommand driveCommand, IIntakeSubsystem intake, IShooterSubsystem shooter,
+      DriveArmCommand driveArmCommand, IClimbSubsystem climb, ClimbControllerCommand climbCommand) {
     super(logger);
-    _autonomousCommand = autonomousCommand;
 
+    _config = config;
     drivetrain.setDefaultCommand(driveCommand);
     intake.setDefaultCommand(driveArmCommand);
     climb.setDefaultCommand(climbCommand);
 
+    m_autonomousCommand = new SequentialCommandGroup(new ShooterStartupCommand(logger, shooter),
+        new TimedAutonMoveBackward(logger, _config, drivetrain).withTime(1.5).withVelocity(1),
+        new TimedAutonMoveBackward(logger, _config, drivetrain).withTime(1).withVelocity(.5).withTimeout(3),
+        new TrackVisionTarget(logger, config, drivetrain, new Limelight(config)).withTimeout(4),
+        new ShootOneBallCommand(logger, shooter), new ShootOneBallCommand(logger, shooter),
+        new ShootOneBallCommand(logger, shooter), new ShooterShutdownCommand(logger, shooter));
+
+    // TODO finish this auton after Glacier Peak.
```

Comment:
Remove this for when you merge to master, keep it in the feature branch.   Keep the Todo stub tho